doc/Doxyfile.in

```diff
@@ -125,7 +125,7 @@ EXAMPLE_PATTERNS       =
 EXAMPLE_RECURSIVE      = NO
 IMAGE_PATH             =
 INPUT_FILTER           =
-FILTER_PATTERNS        =
+FILTER_PATTERNS        = *.py="@FreeOrion_SOURCE_DIR@/doc/tag_filter.py"
```

Comment:
Maybe this helps:

http://stackoverflow.com/a/2757504

You need to dump pass any non-py file content then.